### PR TITLE
add-quota-entity

### PR DESCRIPTION
**Pull request by goldi-tz**
- VCITA-39615

### CODE REVIEW

File: entities/communication/quota.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "type": "object",
+  "properties": {
+    "business_uid": {
+      "type": "string",
+      "description": "The business that the quota relates to."
+    },
+    "quota_uid": {
```
Comment (eliran-vcita) [inline entities/communication/quota.json:8]:
```suggestion
    "uid": {
```

File: entities/communication/quota.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      "type": "string",
+      "description": "The entity's unique identifier."
```
Comment (eliran-vcita) [inline entities/communication/quota.json:10]:
```suggestion
      "description": "The quota's unique identifier."
```

File: entities/communication/quota.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    "remaining_minutes": {
+      "type": "integer",
+      "description": "The remaining minutes for calls."
+    },
+    "source": {
+      "type": "string",
+      "description": "The quota creator."
```
Comment (eliran-vcita) [inline entities/communication/quota.json:18]: is this an enum? By the looks of the example, seems like there is a set of values.

File: entities/communication/quota.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+  "type": "object",
+  "properties": {
+    "business_uid": {
+      "type": "string",
+      "description": "The business that the quota relates to."
```
Comment (nikitarablus) [inline entities/communication/quota.json:6]: Unique identifier of the business that the quota relates to

File: entities/communication/quota.json (hunk 2 above)
Comment (nikitarablus) [inline entities/communication/quota.json:10]: Unique identifier of the quota object

File: entities/communication/quota.json (hunk 3 above)
Comment (nikitarablus) [inline entities/communication/quota.json:18]: The name of the source where the quota was created (?)

File: entities/communication/quota.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      "type": "integer",
+      "description": "The remaining minutes for calls."
+    },
+    "source": {
+      "type": "string",
+      "description": "The quota creator."
+    },
+    "created_at": {
+      "type": "timestamp",
+      "description": "The creation date and time of the quota."
```
Comment (nikitarablus) [inline entities/communication/quota.json:22]: The time the quota was created

File: entities/communication/quota.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    "updated_at": {
+      "type": "timestamp",
+      "description": "The last updated date and time of the quota."
```
Comment (nikitarablus) [inline entities/communication/quota.json:26]: The time the quota was last updated

File: entities/communication/quota.json (hunk 3 above)
Comment (nikitarablus) [reply entities/communication/quota.json:18]: Also, what are the possible values?

File: entities/communication/quota.json (hunk 4 above)
Comment (ramsterita) [reply entities/communication/quota.json:6]: Unique identifier of the BusinessAccount that the quota relates to

**Comment by ramsterita** [comment]
@eliran-vcita @nikitarablus 2 concerns:
1. The name of the entity is very generic. What are you counting here? Is it a voicequota or smsquota.
2. We have the quota manager project. How does this relate to that project? Is it complementary? Is it aligned?

**Comment by goldi-tz** [comment]
> @eliran-vcita @nikitarablus 2 concerns:
> 
> 1. The name of the entity is very generic. What are you counting here? Is it a voicequota or smsquota.
I changed it to entities/communication/voice_call/quota.json
> 2. We have the quota manager project. How does this relate to that project? Is it complementary? Is it aligned?
It's a solution until the quota manager project comes up.
In the future, it should be handled there

**Review by ramsterita — APPROVED** [review]

**Pull request merged**